### PR TITLE
Python-3 compatibility

### LINKED ISSUE #12: Python 3 compatibility (NameError: name 'textfsm' is not defined during installation)
I can install `gtextfsm` using Python 2.7 but not using Python 3.5.

Python 2:
```sh
user@debian:/tmp$ virtualenv py27
New python executable in /tmp/py27/bin/python
Installing setuptools, pip, wheel...done.
user@debian:/tmp$ . ./py27/bin/activate
(py27) user@debian:/tmp$ python
Python 2.7.3 (default, Jun 21 2016, 18:38:19) 
[GCC 4.7.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> 
(py27) user@debian:/tmp$ pip install gtextfsm
Collecting gtextfsm
Installing collected packages: gtextfsm
Successfully installed gtextfsm-0.2.1
```

Python 3:
```sh
user@debian:/tmp$ virtualenv-3.5 -p python3.5 py35
Already using interpreter /home/user/bin/python3.5
Using base prefix '/home/user'
New python executable in /tmp/py35/bin/python3.5
Also creating executable in /tmp/py35/bin/python
Installing setuptools, pip, wheel...done.
user@debian:/tmp$ . py35/bin/activate
(py35) user@debian:/tmp$ pip install gtextfsm
Collecting gtextfsm
  Using cached gtextfsm-0.2.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-wpyy1qn6/gtextfsm/setup.py", line 19, in <module>
        import textfsm
      File "/tmp/pip-build-wpyy1qn6/gtextfsm/textfsm/__init__.py", line 2, in <module>
        __version__ = textfsm.__version__
    NameError: name 'textfsm' is not defined
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-wpyy1qn6/gtextfsm/
```

### PR DESCRIPTION
**Pull request by xavierhardy**
Fixes #12 , thank you @jonathanslenders.

### CODE REVIEW

**Comment by buxtronix** [comment]
Thanks for offering your contribution to Textfsm. A couple of things before we accept it.

1) Please ensure all commit authors provide a CLA, per the information from 'googlebot'.

2) fyi, the commit will be merged as a squash commit (the series of commits will become just one).

Thanks

**Comment by xavierhardy** [comment]
I'll sqash them and a CLA.

**Comment by buxtronix** [comment]
Dont worry, I can address the squash when I accept the pull.

Note that the other contributors to the commits will need a CLA also, assuming their commits get squashed.

**Comment by xavierhardy** [comment]
OK, I signed it.

**Comment by xavierhardy** [comment]
@vincentbernat and @jonathanslenders, could you sign a CLA using the instructions above, please?

Thanks in advance

**Comment by vincentbernat** [comment]
Done.

**Comment by jonathanslenders** [comment]
Signed.

**Comment by xavierhardy** [comment]
@vincentbernat and @jonathanslenders , are you with your commits being contributed to `google/textfsm`?

**Comment by vincentbernat** [comment]
@xavierhardy I am.

**Comment by jonathanslenders** [comment]
@xavierhardy: Yes, that's fine.

**Comment by xavierhardy** [comment]
Thank you both, @buxtronix, can you review the pull request, please?

File: setup.py [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
 
-from distutils.core import setup
+from setuptools import setup
 
 import textfsm
 
 
 setup(name='textfsm',
-      maintainer='Google',
+      maintainer='Google; Jonathan Slenders',
```
Comment (buxtronix) [inline setup.py]: This should just be 'Google' unless explicitly part of the github maintainers.

File: textfsm_test.py [hunk 2]
```diff
@@ -15,13 +15,15 @@
...
 
 
-import cStringIO
+from six import StringIO
+
 import unittest
 # http://python-mock.sourceforge.net/
 
-import textfsm
+import jtextfsm as textfsm
```
Comment (buxtronix) [inline textfsm_test.py]: Should just be 'import textfsm'.

**Comment by xavierhardy** [comment]
There is just a thing to fix, StringIO files need to be closed in Python 3.

**Comment by termlen0** [comment]
I was able to download the repo and install int python 3.5.2 venv. However, pip install gtextfsm is still broken. I assume the push to pyPI is still pending. Appreciate your response.

**Pull request merged**